prometheus.go

```diff
@@ -55,7 +64,22 @@ func StartPrometheusExporter(cfg *PrometheusConfig) error {
 
 	// Next, we'll register all our metrics.
 	prometheus.MustRegister(mailboxCount)
-	prometheus.MustRegister(mailboxReadCount)
+	prometheus.MustRegister(activeSessions)
+	prometheus.MustRegister(standbySessions)
+	prometheus.MustRegister(inUseSessions)
+
+	// Periodically update session classification metrics from internal tracker
+	go func() {
+		ticker := time.NewTicker(10 * time.Second)
+		defer ticker.Stop()
```

Comment:
This Goroutine will never really stop, so the `defer` will never be called. Perhaps we should pass in the `interceptor` that we create in `run()` in `aperture.go`. Then at least we can listen on `interceptor.ShutdownChannel()`.

Reply:
Great catch. I'll refactor this after standup.

Reply:
Pushed a commit that should address this. Also fixed up the comment punctuation across the board.

Will rebase and squash before merge if everything looks ok.